apps/explorer/src/components/SuiTokenCard.tsx

```diff
@@ -1,46 +1,87 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+import { formatAmount } from '@mysten/core';
 import { Sui } from '@mysten/icons';
 
 import { StatsWrapper } from './HomeMetrics/FormattedStatsAmount';
 
+import { useSuiCoinData } from '~/hooks/useSuiCoinData';
 import { Card } from '~/ui/Card';
 import { Heading } from '~/ui/Heading';
+import { Text } from '~/ui/Text';
 
 export function SuiTokenCard() {
+    const { data, isLoading } = useSuiCoinData();
+    const {
+        priceChangePercentageOver24H,
+        currentPrice,
+        totalSupply,
+        fullyDilutedMarketCap,
+    } = data || {};
+
+    const isPriceChangePositive = Number(priceChangePercentageOver24H) > 0;
+    const formattedPrice = currentPrice
+        ? currentPrice.toLocaleString('en', {
+              style: 'currency',
+              currency: 'USD',
+          })
+        : '--';
+
     return (
         <Card bg="lightBlue" spacing="lg">
-            <div className="flex items-center gap-2">
-                <div className="h-4.5 w-4.5 items-center justify-center rounded-full bg-sui p-1">
-                    <Sui className="h-full w-full text-white" />
+            <div className="md:max-lg:max-w-[336px]">
```

Comment:
What does this className mean?, I've not seen it chained like this

Reply:
yeah this looks absolutely crazy LOL, this translates to "for the medium to large breakpoint size, set the max width to 336px" - https://tailwindcss.com/docs/responsive-design#targeting-a-breakpoint-range

I guess this could also be written as `md:max-w-[336px] lg:max-w-none`

Reply:
Ah I see, it's the wedge in between sm and lg styling. Got it, thanks :)